client.go

```diff
@@ -24,6 +24,8 @@ import (
 	"strings"
 	"sync"
 	"time"
+
+	"golang.org/x/net/http2"
```

Comment:
@golgoth31 Thanks for submitting PR. And providing insight into the issue. 
`golang.org/x/net/http2` package is part of the language since go1.6. So externally referring is not required.

I will investigate http2 and address it. Thanks again.